python/lsst/meas/base/forcedPhotImage.py

```diff
@@ -123,6 +174,53 @@ def __init__(self, butler=None, refSchema=None, **kwds):
             self.makeSubtask("applyApCorr", schema=self.measurement.schema)
         self.makeSubtask('catalogCalculation', schema=self.measurement.schema)
 
+    def getInitOutputDatasets(self):
+        return {"outputSchema": lsst.afw.table.SourceCatalog(self.measurement.schema)}
+
+    def adaptArgsAndRun(self, inputData, inputDataIds, outputDataIds, butler):
+        # ForcedPhotImageTask.run() expects to work in-place on measCat. This requires it to be
+        # constructed here before being passed as an input to run(). It should not be in the
+        # inputData dict(), but check that we don't clobber something.
```

Comment:
If it shouldn't be there but is, that should probably raise an exception.

Reply:
Running last minute tests shows I needed to be a bit clearer in the comment (for myself if nothing else).  If measCat doesn't exist, it should be made, and for the ForcedPhotCoadd case, which just calls ForcedPhotImageTask, we do not expect it to exist.  However, for ForcedPhotCcd, we construct the measCat there, and then pass it up to the parent class in adaptArgsAndRun.  In this case, it does exist, and shouldn't be made.